laravel/tests/Feature/AdministrationSystemstatusTest.php

```diff
@@ -42,4 +42,14 @@ public function test_page_gets_displayed_when_authenticated(): void
         $response->assertStatus(200);
         $response->assertViewIs('administration.systemstatus');
     }
+
+    /**
+     * Test, that the systemstatus page can be displayed.
+     */
+    public function test_view_systemstatus_gets_displayed(): void
```

Comment:
[IMPROVEMENT]

Give the test a proper name, so that it's understandable in the output of the testing:
```suggestion
    public function test_systemstatus_view_gets_displayed(): void
```